olp-cpp-sdk-authentication/include/olp/authentication/AuthenticationCredentials.h

```diff
@@ -27,68 +27,96 @@
-   * interpreted as a sequence of characters.
+   * interpreted as a sequence of characters and retrieves a value with your
+   * credentials.
    *
    * The stream must contain the following sequences of characters:
-   * * here.access.key.id = <your access key ID>
-   * * here.access.key.secret = <your access key secret>
+   * * here.access.key.id &ndash; your access key ID
+   * * here.access.key.secret &ndash; your access key secret
    *
    * @param[in] stream The stream from which the credentials are read.
+   *
    * @return An optional value with your credentials if the credentials were
    * read successfully, or no value in case of failure.
    */
   static boost::optional<AuthenticationCredentials> ReadFromStream(
       std::istream& stream);
 
-  /*
-   * @brief Parses the **credentials.properties** file downloaded from the OLP
-   * [website](https://developer.here.com/olp/documentation/access-control/user-guide/topics/get-credentials.html)
-   * and retrieves a value with your credentials.
+  /**
+   * @brief Parses the **credentials.properties** file downloaded from the [Open
+   * Location Platform (OLP)](https://platform.here.com/)
+   * website and retrieves a value with your credentials.
    *
    * The file must contain the following lines:
-   * * here.access.key.id = <your access key ID>
-   * * here.access.key.secret = <your access key secret>
+   * * here.access.key.id &ndash; your access key ID
+   * * here.access.key.secret &ndash; your access key secret
+   *
+   * For instructions on how to get the **credentials.properties** file, see
+   * the [Get
+   * Credentials](https://developer.here.com/olp/documentation/access-control/user-guide/topics/get-credentials.html)
+   * section in the Terms and Permissions User Guide.
    *
    * @param[in] filename The path to the file that contains the credentials.
    * An empty path is replaced with the following default path:
-   * $HOME/.here/credentials.properties
-   * @return An optional value with your credentials if the credentials were
+   * `$HOME/.here/credentials.properties`
+   *
+   * @return The optional value with your credentials if the credentials were
    * read successfully, or no value in case of failure.
    */
   static boost::optional<AuthenticationCredentials> ReadFromFile(
       std::string filename = {});
   AuthenticationCredentials() = delete;
 
   /**
-   * @brief Constructor
-   * @param key The client access key identifier.
-   * @param secret The client access key secret.
+   * @brief Creates the `AuthenticationCredentials` instance with your access
+   * key ID and access key secret.
+   *
+   * @param key Your access key ID.
+   * @param secret Your access key secret.
    */
   AuthenticationCredentials(std::string key, std::string secret);
 
   /**
-   * @brief Copy Constructor
-   * @param other object to copy from
+   * @brief Creates the `AuthenticationCredentials` instance that is a copy
+   * of the `other` parameter.
```

Comment:
```suggestion
   * of the `other` credentials.
```
Reason - the parameter is too generic

Reply:
Disagree: `other` is not the name of the credentials. It's the name of the parameter that contains the credentials.

Reply:
Changed to "the `other` parameter that contains the access credentials".